sig-scalability/slos/watch_latency.md

```diff
@@ -0,0 +1,17 @@
+## Watch latency SLI details
+
+### User stories
+- As an aministrator, if Kubernetes is slow, I would like to know if the root
+cause of it is slow api-machinery (slow watch) or something farther the path
+(lack of network bandwidth, slow or cpu-starved controllers, ...)
+
+### Other notes
+- Pretty much all control loops in Kubernetes are watch-based. As a result
+slow watch means slow system in general.
+- Note that how we measure it silently assumes no clock-skew in case of
+cluster with multiple masters.
+
+### TODOs
```

Comment:
Btw - are we not having a test scenario here (like for other ones)?

Reply:
We don't have test cases for SLIs - test cases make sense only for things for which we have SLO.

If we will have an SLO for that in the future, there should be a test case.

Reply:
Yeah, that makes sense. Thanks.